onnxruntime/test/xctest/ortxctest.m

```diff
@@ -20,16 +20,4 @@ - (void)tearDown {
     // Put teardown code here. This method is called after the invocation of each test method in the class.
 }
 
-- (void)testExample {
-    // This is an example of a functional test case.
-    // Use XCTAssert and related functions to verify your tests produce the correct results.
-}
-
-- (void)testPerformanceExample {
```

Comment:
these tests don't do anything useful. moreover, testPerformanceExample takes a non-trivial amount of time to run